types/abi/src/standard.rs

```diff
@@ -41,7 +41,9 @@ pub fn get_sfx_transfer_abi() -> SFXAbi {
         ingress_abi_descriptors: PerCodecAbiDescriptors {
             // assume all indexed in topics ("+")
             for_rlp: b"Transfer:Log(from+:Account20,to+:Account20,amount+:Value128)".to_vec(),
-            for_scale: b"Transfer:Enum(from:Account32,to:Account32,amount:Value128)".to_vec(),
+            for_scale:
+                b"Transfer:Enum(palletIndex:Byte,from:Account32,to:Account32,amount:Value128)"
```

Comment:
can't see palletIndex being handled anywhere so maybe remove for now and I'll fix in separate PR

Reply:
I need it to decode correctly. Otherwise the other fields are shifted by one byte

Reply:
that's covered by Abi Recoding. at assumes Enum consumes 1 byte as prefix memo. see types/abi/src/standards test